.gitignore

```diff
@@ -9,3 +9,4 @@
 .project
 /.settings/
 enchcracker.log
+*.dll
```

Comment:
Could you leave this unneeded change to the gitignore out? There shouldn't be a DLL in the repo anyway.